samples/tfm/tfm_psa_template/README.rst

```diff
@@ -153,15 +153,15 @@ Application and TF-M firmware update
 ====================================
```

Comment:
I can't verify that the changes of this PR to this sample is correct, because something is broken in main branch on signing keys.